scripts/tablegen.py

```diff
@@ -37,13 +46,17 @@ def endOfTheList(context, lineNum, lastLineNum, totalNum):
     if lineNum != lastLineNum:
         return False, valueList
 
+
     for nextLineNum in range(lineNum + 1, totalNum):
         nextLine = context[nextLineNum]
+
         if  len(nextLine.lstrip()) != 0 and '#' != nextLine.lstrip()[0] and ':' in nextLine:
             if whitespaces >= (len(nextLine) - len(nextLine.lstrip())) / 2:
                 if flag == 0:
                     valueList.append(currentLine.split(':', 1)[1].strip())
                 return True, valueList
+            else:
```

Comment:
if this renders correctly, then the whitespace comparison is not needed and you should be able to delete line 54/58/59

Reply:
Note iirc, the reason for this whitespace comparison was to produce a description field - but my memory is fuzzy. @gyliu513 may know more.

Reply:
@sdake without that if-else statement it doesn't render correctly. It only shows the content from the values.yaml under istio and not the values from the sub charts dir.

Reply:
oh right that makes sense.